lib/opensearch/transport/client.rb

```diff
@@ -191,6 +190,32 @@ def perform_request(method, path, params = {}, body = nil, headers = nil)
         transport.perform_request(method, path, params, body, headers)
       end
 
+      def perform_ping_request(*args)
```

Comment:
I think these can be used publicly, do we want it to be `ping` or `ping!`, `head`, `delete`?

Reply:
They are special cases of `perform_request` where instead of returning the response body, they return a boolean (for ping and exists API methods like `indices.exists`), and support `ignore: [404]` for delete requests on resources that have already been deleted.

Just like the original `perform_request` method, we dont expect the user to use them. We already provide an Http namespace custom requests.